script/uninstall

```diff
@@ -0,0 +1,15 @@
+#! /bin/sh
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+echo "=> Stopping shutdown listener...\n"
```

Comment:
We're going to need something like this as well: `sudo update-rc.d listen-for-shutdown.sh remove`. I haven't tested this line but I think it's right.

Reply:
Ok I figured, want me to add that or can you do it when you test?